src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/convenience/key_vault_credential.py

```diff
@@ -0,0 +1,26 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+#pylint: skip-file
+
+import adal
+import urllib
+
+from msrest.authentication import authentication
+
+from . import httpbearerchallenge
+from . import httpbearerchallengecache as challengecache
+
+class keyvaultcredential(authentication):
```

Comment:
It looks like I must have pressed a funky shortcut in VS because it seems to have made this file all lowercase. Currently it's not used by anything (though it may be used in the convenience layer auth later on).